clients/cobol-lsp-vscode-extension/src/services/Settings.ts

```diff
@@ -419,4 +465,30 @@ export class SettingsService {
       .get(SETTINGS_CPY_NDVR_DEPENDENCIES);
     return setting;
   }
+
+  private static getUnreachableCodeSeverityValue(): string {
+    return (
+      vscode.workspace
+        .getConfiguration()
+        .get(SETTINGS_UNREACHABLE_CODE_SEVERITY) ?? NONE
+    );
+  }
+
+  private static getCopybookConfigValues(
```

Comment:
Please move it back where it was, there is no reason to create extra merge conflicts for others.